toolbox/GEE_Connector.pyt

```diff
@@ -919,6 +951,14 @@ class AddImgCol2MapbyID:
         if parameters[1].valueAsText:
             arcgee.data.check_start_date(parameters[1])
 
+        # Check if the JSON file name contains spaces or special characters.
+        json_path = parameters[10].valueAsText
+        if json_path:
+            if arcgee.data.has_spaces_or_special_chars(json_path):
+                parameters[10].setErrorMessage(
+                    "The JSON file name contains spaces or special characters."
```

Comment:
Missing a space after the period/full stop. This will give: `characters.Please specify`

And same below.